extensions/kafka/src/main/java/io/deephaven/processor/ObjectProcessor.java

```diff
@@ -66,6 +66,22 @@ static <T> ObjectProcessor<T> rowLimited(ObjectProcessor<T> delegate, int rowLim
         return ObjectProcessorRowLimited.of(delegate, rowLimit);
     }
 
+    /**
+     * Creates a "no-operation" object processor that ignores the input object chunk. If {@code fillWithNullValue} is
+     * {@code true}, during {@link ObjectProcessor#processAll(ObjectChunk, List) processAll}
+     * {@link WritableChunk#fillWithNullValue(int, int) fillWithNullValue} will be invoked on each output chunk;
+     * otherwise, the output chunk contents will not be modified. In either case, the processing will increment the
+     * output chunks sizes.
+     *
+     * @param outputTypes the output types
+     * @param fillWithNullValue if the output chunks should be filled with the appropriate null value;
+     * @return the no-op object processor
+     * @param <T> the object type
+     */
+    static <T> ObjectProcessor<T> noop(List<Type<?>> outputTypes, boolean fillWithNullValue) {
```

Comment:
Does this belong in production sources? Same question for `ObjectProcessorNoop`, of course.

Reply:
I think yes - it's potentially useful scaffolding to test out a given publisher before you have your own ObjectProcessor impl.

For example, let's say I want to develop my own Capn Proto ObjectProcessor, but in parallel I still want to allow other developers to work on the downstream queries while the impl is still in progress - they can become "unblocked" by faking it w/ noop:

```groovy
my_table = consumeToTable(..., objectProcessorSpec(noop([Type.stringType(), Type.intType()], true), ["Name", "Age"]))

Reply:
That seems pretty tenuous.